install-template.sh

```diff
@@ -109,6 +109,20 @@ else
   configure_cache
   test_cache_configured || (echo "Failed to configure the garnix cache, cancelling installation." && exit 1)
   echo The garnix cache is now configured.
+else
+  echo "For '@@toolName@@' to work well, Nix needs to use the garnix.io binary"
+  echo "cache. But the cache configuration cannot be found."
+  echo
+  echo "This installer attempted to add the cache config to /etc/nix/nix.conf,"
+  echo "but that file was not writable (you may be using NixOS)."
+  echo "Please add the following lines to your Nix config and restart your nix-daemon:"
+  echo "    extra-substituters = https://cache.garnix.io"
+  echo "    extra-trusted-public-keys = cache.garnix.io:CTFPyKSLcx5RMJKfLo5EEPUObbA78b0YQ2DTCJXqr9g="
+  echo
+  echo "On NixOS you can add these to nix.extraOptions in /etc/nixos/configuration.nix."
```

Comment:
```suggestion
  echo "On non-NixOS systems you can add these lines to /etc/nix/nix.conf."
  echo "On NixOS you can add these to nix.extraOptions in /etc/nixos/configuration.nix."
```
This is a bit weird advice, since that file is not writeable. Although users may be able to change that. Although it's also mentioned above that that's where the config goes. :shrug: TIOLI.